contracts/cw721-base/src/state.rs

```diff
@@ -18,6 +19,9 @@ pub struct TokenInfo {
     pub description: String,
     /// A URI pointing to an image representing the asset
     pub image: Option<String>,
+
+    /// You can add any custom metadata here when you extend cw721-base
+    pub extension: T,
```

Comment:
This is great! Not sure if description / image are needed anymore if there is an extension?

Much more usable like this though. :pray: